homeassistant/components/bond/bond_entity.py

```diff
@@ -0,0 +1,38 @@
+"""An abstract class common to all Bond entities."""
```

Comment:
you can remove the bond_ prefix on file name. All the integrations ive seen just call it entity.py

Reply:
got it, fixed